### DISCUSSION TITLE
Discrepency between speechscript and audio

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#177] [General]
In the English speech script I find ths otherwise unused dialog snippet.
` [A1_M1302_MEE_4529] 
{Be careful.} `

The file is not otherwise used and without listening to it, I tried to insert it where it seems to go in response to Marlow 
saying he's found the beacon.   Surprise, surprise Meeks says "any sign of the Nostromo crew?" with a slight bit of sarcasm.  
Not "be careful." as the speechscript shows.   HAHA! Oh well.

**Comment by MattFiler** [comment]
Yeah, strange - I just listened to the audio file and you're correct, the line is "any sign of the Nostromo crew". I assume since it's unused it was probably changed a couple times and just not updated properly. That line isn't labelled to any other entries either.

**Reply by ghramsey** [reply]
Yeah.  That makes sense.  Imagine my surprise when I heard it.   But I did add the other dialog.  
Marlow saying his head is buzzing and he feels like he's going to puke.    I wonder why that was deleted? 

It seems logical that he'd be feeling sick that close to a massive broadcast.  Anyway, I did add it back.
 I need to tweak the delays though.  He's saying it all at once.
